grafana-plugin/src/components/AlertTemplates/AlertTemplatesForm.tsx

```diff
@@ -113,11 +107,7 @@ const AlertTemplatesForm = (props: AlertTemplatesFormProps) => {
 
   const getGroupByTemplateName = (templateName: string) => {
     Object.values(groups).find((group) => {
-      const foundTemplate = group.find((obj: any) => {
-        if (obj.name == templateName) {
-          return obj;
-        }
-      });
+      const foundTemplate = group.find((obj) => obj.name === templateName);
```

Comment:
the first argument to `find` [should be a predicate](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) (ie. function that returns a `boolean`. The previous version returned `Template | undefined`